mock-data/connectPost.js

```diff
@@ -0,0 +1,20 @@
+const pg = require('pg');
+const config  = {
+	database: 'booking',
+	port: 5432,
+}
+
+const pool = new pg.Pool(config);
+
+const queryHandler = function(q) {
+	pool.connect((err, client, done) => {
+		if(err) {
```

Comment:
instead of console log should throw error I think. Also below you use ternary statement for error handling, but in this one you use a if statement,.